test/token/WstEthL2TokenHarness.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache 2
+pragma solidity >=0.8.8 <0.9.0;
+
+import {WstEthL2Token} from "src/token/WstEthL2Token.sol";
+
+contract WstEthL2TokenHarness is WstEthL2Token {
+    function exposed_authorizeUpgrade(address _newImplementation) external view {
```

Comment:
What's the need for this vs just calling `upgradeToAndCall` which is already public?

Reply:
Since the `_authorizeUpgrade` method is overridden to add an `onlyOwner` modifier, this test enables us to verify that isolated logic, while the `upgradeToAndCall` method is gated by the `onlyProxy` modifier.

This test should cover the actual proxy upgrade logic: `testFuzz_PerformsAndInitializesAnUpgradeThatAddsNewFunctionalityToTheToken`